server/lib/handlebars.js

```diff
@@ -152,6 +152,30 @@ handlebars.registerHelper('encodeURIComponent', str => {
 
 handlebars.registerHelper('formatCurrencyObject', (obj, props) => formatCurrencyObject(obj, props.hash));
 
+handlebars.registerHelper('formatOrderSummary', order => {
+  if (order.currency === null || order.totalAmount === null) {
+    return null;
+  }
+
+  const currencyObj = new Object();
+  currencyObj[order.currency] = order.totalAmount;
+
+  let ret = formatCurrencyObject(currencyObj);
```

Comment:
Simpler alternative using `formatCurrency` from `server/lib/utils.js` + explicit variable name:

```suggestion
  const formattedAmount = formatCurrency(order.totalAmount, order.currency);
```